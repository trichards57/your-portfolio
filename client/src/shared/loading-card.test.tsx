import React from "react";
import ReactDOM from "react-dom";
import LoadingCard from "./loading-card";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");

  ReactDOM.render(<LoadingCard />, div);
});

it("renders correctly", () => {
  const tree = renderer.create(<LoadingCard />).toJSON();

  expect(tree).toMatchSnapshot();
});
