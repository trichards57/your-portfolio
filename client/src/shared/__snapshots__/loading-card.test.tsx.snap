// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="MuiCardContent-root"
  >
    <p
      className="MuiTypography-root makeStyles-date-1 MuiTypography-body1 MuiTypography-gutterBottom"
    >
      <span
        className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      />
    </p>
    <h2
      className="MuiTypography-root makeStyles-trimText-2 MuiTypography-h5"
    >
      <span
        className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      />
    </h2>
    <p
      className="MuiTypography-root makeStyles-location-3 MuiTypography-body1 MuiTypography-colorTextSecondary"
    >
      <span
        className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      />
    </p>
    <p
      className="MuiTypography-root MuiTypography-body2"
    >
      <span
        className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      />
    </p>
    <p
      className="MuiTypography-root MuiTypography-body2"
    >
      <span
        className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      />
    </p>
  </div>
</div>
`;
